packages/turbo-graph-ui/lib/turbo.ts

```diff
@@ -62,14 +62,22 @@ async function executeCommand(
 
 async function findTurboConfig(currentPath = '.'): Promise<Data> {
   const files = await fs.readdir(currentPath)
-  const turboConfig = files.find((file) => file === 'turbo.json')
+  const turboConfig = files.find((file) => file === 'turbo.json' || file === 'turbo.jsonc')
   const continueSearch = () => findTurboConfig(`..${path.sep}${currentPath}`)
   if (!turboConfig) {
     return continueSearch()
   }
   const file = `${currentPath}${path.sep}${turboConfig}`
-  const buffer = await fs.readFile(file)
-  const config = JSON.parse(buffer.toString())
+  const contents = await fs.readFile(file, { encoding: 'utf8' })
```

Comment:
i'm getting the contents as a string here so that, if JSON.parse fails, the buffer doesn't have to be stringified a second time. `buffer.toString()` used utf8 encoding anyway (the default), and json is specced to always be utf-8 encoded.